### PR TITLE
STORM-1436: Set Travis Heap size to fit in memory limits.

### PR DESCRIPTION
**Pull request by revans2**

### CODE REVIEW

**Comment by revans2** [comment]
There is a separate test on travis that consistently fails on JDK7.  I am looking into it, but it should probably be on a separate pull request.

**Comment by redsanket** [comment]
LGTM +1

File: dev-tools/travis/travis-script.sh [hunk 1]
```diff
@@ -24,8 +24,10 @@ cd ${STORM_SRC_ROOT_DIR}
 
 # We should be concerned that Travis CI could be very slow because it uses VM
 export STORM_TEST_TIMEOUT_MS=150000
+# Travis only has 3GB of memory, lets use 1GB for build, and 1.5GB for forked JVMs
+#export MAVEN_OPTS="-Xmx1024m"
```
Comment (HeartSaVioR) [inline dev-tools/travis/travis-script.sh]: Why do you explain with commenting at all?

**Comment by HeartSaVioR** [comment]
+1 though I'd like to see the reason about comments in travis-script.sh.

File: dev-tools/travis/travis-script.sh (hunk 1 above)
Comment (revans2) [inline dev-tools/travis/travis-script.sh]: oops I should not have commented that out.  Looks like it works without it, but I meant to not have it in a comment.  will fix it.

**Comment by revans2** [comment]
I fixed my issue with having the HEAP commented out, and I added in a heap setting to the clojure tests in storm-core.  I suspect that it is the same issue that is causing failures on JDK7

**Comment by revans2** [comment]
On my travis build all of the tests passed, except for one test with kafka that is not related.  It looks like all of the failures were the OOM killer kicking in.

**Comment by revans2** [comment]
The test failure is unrelated.

**Comment by knusbaum** [comment]
+1 Great Jorb!

**Comment by HeartSaVioR** [comment]
Great, thanks @revans2

**Pull request merged**